voxelate/voxelate.py

```diff
@@ -480,15 +581,20 @@ def __get_voxels_condition_list(self):
                     
                     is_roof = bool(x) and not bool(voxel_3d_list[next_zi][yi][xi])
                     is_exterior = bool(x) and any([not prev_x, not next_x, not back_x, not frnt_x])
-                    is_exterior_corner = bool(x) and [prev_x, next_x, back_x, frnt_x].count(False) >= 2
+                    is_exterior_corner = bool(x) and [prev_x, next_x, back_x, frnt_x].count(False) == 2
+                    is_exterior_corner_u = bool(x) and [prev_x, next_x, back_x, frnt_x].count(False) == 3
+                    is_exterior_corner_o = bool(x) and [prev_x, next_x, back_x, frnt_x].count(False) == 4
```

Comment:
change `[prev_x, next_x, back_x, frnt_x].count(False)` line to variable `exterior_count`